### PR TITLE
removed screen for running version with JSServe

### PR DESCRIPTION
**Pull request by TabeaW**
With this Tyler works smoothly with JSServe (#47). The only problem might be the for-loops which are now true permanently.

### CODE REVIEW

**Comment by TabeaW** [comment]
```
using JSServe
using Tyler,Extents,WGLMakie
server=JSServe.Server("0.0.0.0", 8082)
extent=Extent(Y=(46.18,55.78),X=(3.088,17.112)) #Germany
tyler=Tyler.Map(extent)
route!(server, "/browser-display" => App(tyler.figure))
```
will show the tyler map in the JSServe App.

File: src/Tyler.jl [hunk 1]
```diff
@@ -134,12 +131,12 @@ function Map(extent, extent_crs=wgs84;
...
         max_parallel_downloads, OrderedSet{Tile}(),
         tiles_being_added, downloaded_tiles,
-        display_task, download_task, screen,
+        display_task, download_task, 
         depth, halo, scale
     )
     tyler.zoom[] = get_zoom(tyler, extent)
     download_task[] = @async begin
-        while isopen(screen)
+        while true
```
Comment (rafaqz) [inline src/Tyler.jl:139]: Does this mean the task runs forever now?

File: src/Tyler.jl [hunk 2]
```diff
@@ -150,14 +147,10 @@ function Map(extent, extent_crs=wgs84;
         empty!(tyler.displayed_tiles)
     end
     display_task[] = @async begin
-        while isopen(screen)
+        while true
```
Comment (rafaqz) [inline src/Tyler.jl:150]: Same here

**Review by rafaqz — COMMENTED** [review]
Thanks. Im mostly concerned that this change means we are spawning tasks that never break  out of their while loops.

File: src/Tyler.jl (hunk 1 above)
Comment (TabeaW) [reply src/Tyler.jl:139]: That's the big question, I am not quite sure about that, but had no real alternative without the screen in mind.

Comment (rafaqz) [reply src/Tyler.jl:139]: My guess is this task will just keep running every 0.01 seconds until you kill the session, unless it hits another error from the Makie screen not being there anymore (but I think it wont?).

We can't really do that... we need to find another way of breaking the loop.

Comment (rafaqz) [reply src/Tyler.jl:139]: Maybe try prining to stdout in the task and see what happens

**Comment by SimonDanisch** [comment]
I think I should add a function to makie a la `waiting_isopen` that waits until the scene/figure actually gets displayed and only then starts returning true/false! I think only with that we can have a clean condition for those loops

**Comment by SimonDanisch** [comment]
Implemented in https://github.com/MakieOrg/Tyler.jl/pull/69

**Pull request closed without merging**